### PR TITLE
postfix and removes inputFieldWidth

### PR DESCRIPTION
**Pull request by jpye-finch**
updates weight postfix

image

### CODE REVIEW

**Review by Zangetsu101 — APPROVED** [review]
The changes look good. We just need to remember when merging this that a PR needs to be created in countryconfig with the type change (and potentially the form changes)

File: src/form/birth/optional-fields.ts [hunk 1]
```diff
@@ -61,12 +61,8 @@ export const weightAtBirth: SerializedFormField = {
       parameters: [0, 6]
     }
   ],
-  postfix: 'Kg',
-  mapping: getFieldMapping(
-    'weightAtBirth',
-    certificateHandlebars.weightAtBirth
-  ),
-  inputFieldWidth: '78px'
+  postfix: 'kilograms (kg)',
```
Comment (naftis) [inline src/form/birth/optional-fields.ts:64]: we might need to translate this via `formMessageDescriptors` ?

Comment (Zangetsu101) [reply src/form/birth/optional-fields.ts:64]: currently we only support strings for this prop, adding intl message support could be a good addition

**Comment by Zangetsu101** [comment]
Closing in favor of https://github.com/opencrvs/opencrvs-countryconfig/pull/201

**Pull request closed without merging**